### PR TITLE
Descriptions for workflows

### PR DESCRIPTION
**Pull request by decodingahmed**

### CODE REVIEW

**Review by keijoraamat — APPROVED** [review]

File: .github/workflows/jobs-create-env-mock-data.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+#
+# Create mock data workflow
+# 
+# This workflow creates mock institutions and participants for the given envrionment
```
Comment (guypritchard) [inline .github/workflows/jobs-create-env-mock-data.yml:4]:
```suggestion
# This workflow creates mock institutions and participants for the given environment 
```

File: .github/workflows/jobs-create-env-mock-data.yml [hunk 2]
```diff
@@ -1,3 +1,10 @@
+#
+# Create mock data workflow
```
Comment (guypritchard) [inline .github/workflows/jobs-create-env-mock-data.yml:2]: I don't think this is really creating 'mock' data (and is probably beyond the scope of this PR - so pls ignore)   This script merely creates the Participants to allow the system to function with the deployed components.

File: .github/workflows/jobs-integration-test.yml [hunk 3]
```diff
@@ -1,3 +1,12 @@
+#
+# Integration test workflow
+# 
+# This workflow executes integration tests against deployed infrastructure and code to
+# validate that it is functioning correctly. 
+#
+# Integrating tests in the 'main' branch of the 'Tests' repository are executed by default.
```
Comment (guypritchard) [inline .github/workflows/jobs-integration-test.yml:7]: I quite like the term 'integrating tests' - but should it be 'Integration tests'
```suggestion
# Integration tests in the 'main' branch of the 'Tests' repository are executed by default.
```

File: .github/workflows/reusable-helm-package-upload.yml [hunk 4]
```diff
@@ -1,3 +1,13 @@
+#
+# Helm package upload reusable workflow
+# 
+# This workflow tears down an environment of the given name.
+#
+# It requires the environment to have been originally deployed via terraform as it
```
Comment (guypritchard) [inline .github/workflows/reusable-helm-package-upload.yml:6]: This looks like a slight copy/paste accident.

**Review by guypritchard — APPROVED** [review]
Super minor picking of nits.

**Pull request merged**